tests/unit/DateTime/DifferenceTest.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+namespace Coduo\PHPHumanizer\Tests\DateTime;
+
+use PHPUnit_Framework_TestCase;
+
+/**
+ * @author       Ezra Pool <ezra@tsdme.nl>
+ */
+class DifferenceTest extends PHPUnit_Framework_TestCase
+{
+    /**
+     * @test
+     * @dataProvider differenceDataProvider
+     *
+     * @param $expected
+     * @param $date1
+     * @param $date2
+     * @param $instance
+     * @param $is_past
+     */
+    public function it_calculates_differences_correctly($expected, $date1, $date2, $instance, $is_past )
```

Comment:
Could you please rename those methods in order to include "test" word at the beggining? 
For example: 
``public function test_calculating_differences_between_two_dates($expected, $firstDate, $secondDate, $instance, $isPast )``